packages/sdk-middleware-http/src/http.js

```diff
@@ -40,6 +40,13 @@ function calcDelayDuration(
   return retryDelay
 }
 
+function maskAuthData(request: Object) {
+  if (request.headers.authorization)
```

Comment:
Do we still have both cases `authorization` and `Authorization`?

Reply:
Yup. I checked with the team and they are both used everywhere in the code so it makes sense to check/fix both of them.